### DISCUSSION TITLE
How can I format on save (with prettier) **only** if .prettierrc exists in my project?

### DISCUSSION THREAD

**Discussion by malssid** [LazyVim/LazyVim#3115] [Q&A] [answered]
I'm pretty new to neovim/lazyvim. Moved from VSCode and looking for some sort of flag or script that only formats with prettier if the config file is present? VSCode has a 'Require confiG' setting for their prettier extension:
![image](https://github.com/LazyVim/LazyVim/assets/67285754/0aa5cc20-eb49-4cff-a77f-c8f1d4279744)

I've already added prettier to my imports in lazy.lua.
![image](https://github.com/LazyVim/LazyVim/assets/67285754/38c71b4a-39e9-4e12-ba1c-d27e3feaccd2)

**Comment by dpetka2001** [comment, marked as answer]
Haven't done this myself, but there's an example in the [docs](https://www.lazyvim.org/plugins/formatting) about `dprint` to only format when there's a `dprint.json` available, so you might want to follow that example to do the same with `prettier`.

**Reply by malssid** [reply]
Thanks @dpetka2001 . I'll try that and lyk!

**Reply by malssid** [reply]
yup that worked! thanks @dpetka2001
